src/App.tsx

```diff
@@ -1,7 +1,36 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  keyPressed: string;
+};
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    keyPressed: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeypress);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeypress);
+  }
+
+  handleKeypress = (event: KeyboardEvent) => {
+    this.setState({ keyPressed: event.key });
```

Comment:
let's check if `event.code` is `Space` and write `space` if so 😉 
<img width="199" alt="image" src="https://user-images.githubusercontent.com/55802630/182606073-0dca8acd-058f-4491-89fb-0bcfe743f335.png">

Reply:
OK, no problem